File: javascript/problem_template.js
#!/usr/bin/env node

function printBlurb() {
    var blurb = `
    blurb
    `
    console.log(blurb)
}

function problem_xxx(x) {
    return x
}

printBlurb()
console.time('Completed in')
result = problem_xxx(1)
console.log(`Result: ${result}`)
console.timeEnd('Completed in')

File: python/problem_template.py
#!/usr/bin/env python3
import time


def problem_output(func, *args):
    def wrapper(*args):
        blurb()
        start = time.time()
        result = func(*args)
        time_to_end = time.time() - start
        print(f"Result: {result}")
        print(f"Completed in {time_to_end}")

    return wrapper


def blurb():
    print(
        """
    blurb
    """
    )


@problem_output
def problem():
    return 0


def main():
    problem()


if __name__ == "__main__":
    main()
